content/appendices/templates.md

```diff
@@ -47,20 +47,30 @@ The package includes all the following forms of documentation:
 Readme requirements
 The package meets the readme requirements below:
 
-- [ ] Package has a README.md file in the root directory. 
+- [ ] Package has a user-friendly README.md file in the root directory. 
```

Comment:
I hesitate to add qualitative descriptors without defining a bit of what we mean by that. As an editor, it is tricky to help facilitate discussions where there is a disagreement about the "user-friendliness" of a doc, as it can start coming down to opinions. In my opinion, the description below of what the README should include is quite matter-of-fact, so it is easier to make decisions on. Just my 2 cents :)

Reply:
I agree.

Also, IMO, READMEs are a dev/power user resource and not a user documentation. Users need fully developed docs while README is the quick "this is how to install and start hacking on it" kind of thing. BTW, most users do not even find the README b/c they do not interact with the package via the source or the repo. (Not to mentioned that most packages do not ship the README with the source!)

Note that requiring complex doc-like README we may force people to have redundant info on docs and README that may go stable very quickly and actually create more harm than good. It is not hard to find updated docs and stale READMEs b/c of that.

Reply:
ok! so remove the `user-friendly` language? i am fine with that and hear you @ocefpaf about the potential for redundancy as well!!